### PR TITLE
xontrib-sh

### PR DESCRIPTION
**Pull request by anki-code**

### CODE REVIEW

File: docs/bash_to_xsh.rst [hunk 1]
```diff
@@ -138,3 +138,16 @@ there is `bashisms xontrib <https://xon.sh/xontribs.html#bashisms>`_:
     >>> !!
     echo
 
+Work with bash snippets? Take a look at `xontrib-sh <https://xon.sh/xontribs.html#sh>`_:
```
Comment (scopatz) [inline docs/bash_to_xsh.rst:141]: I am a little uncomfortable with the syntax here and should probably move this into the xontrib description

Comment (anki-code) [reply docs/bash_to_xsh.rst]: text removed, examples combined

File: docs/bash_to_xsh.rst [hunk 2]
```diff
@@ -137,4 +138,12 @@ there is `bashisms xontrib <https://xon.sh/xontribs.html#bashisms>`_:
     echo
     >>> !!
     echo
+    >>> xontrib load sh
+    >>> $AND = 'a' + 'nd'
+    >>> # Run bash script:
+    >>> ! for i in {1..3}
+          do
+             echo -n "$AND $i "
+          done
```
Comment (scopatz) [inline docs/bash_to_xsh.rst:147]: This block should go in the description for the xontrib.  Not in the translation guide here

Comment (anki-code) [reply docs/bash_to_xsh.rst]: removed

File: docs/bash_to_xsh.rst [hunk 3]
```diff
@@ -128,13 +128,5 @@ to set :ref:`$XONSH_TRACE_SUBPROC <xonsh_trace_subproc>` to ``True``:
...
-
-.. code-block:: console
-
-    >>> xontrib load bashisms
-    >>> echo echo
-    echo
-    >>> !!
-    echo
-
+there is `bashisms <https://xon.sh/xontribs.html#bashisms>`_
```
Comment (scopatz) [inline docs/bash_to_xsh.rst:131]: "there is" -> "there are the"

Comment (anki-code) [reply docs/bash_to_xsh.rst]: back to school, thanks )

Comment (scopatz) [reply docs/bash_to_xsh.rst]: haha of course, happy to help :wink:

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**